README.md

```diff
@@ -1,16 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Hello Microverse
+# Chopin Concerts
 
-> This project is the main page of a portfolio in mobile version first.
+> This project is a website for the Chopin concerts, perform by a lot of professional pianist, you will be able to see all the information for a concert that will be taking place in 2023.03.18 - 19, from the performers to the pieces.
 
 ## Built With
 
-- html, css
+- html, css and javaScript
 
 ## Live Demo
 
-[Live Demo Link](https://shinhyobelok.github.io/Portfolio-setup-and-mobile-first/)
+[Live Demo Link]()
```

Comment:
- [ ] you have removed the live demo link please make sure to re-add it in order to match the requirements